packages/story-editor/src/components/floatingMenu/elements/shared/useUpdateSelected.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback } from '@googleforcreators/react';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../../app';
+
+function useUpdateSelected() {
```

Comment:
This hook seems a little redundant. Seems like a lot of extra code to switch a function from named params to not. I feel like this also adds a little opaqueness to someone coming onto the codebase when we could just use the action directly. Thoughts?

Reply:
It was just to simplify the usage of the action in a lot of components that are going to use this and make things more DRY. So rather than doing all of this in every component:

```js
  const { updateSelectedElements } = useStory(
    ({ actions: { updateSelectedElements } }) => ({ updateSelectedElements })
  );
  const onClick = () => {
    ...
    updateSelectedElements({ properties });
  }
```

It would be just this:

```js
  const updateSelectedElements = useUpdateSelected();
  const onClick = () => {
    ...
    updateSelectedElements(properties);
  }
```

The difference between named arguments and positional was actually not the primary purpose. It was to DRY up the whole selector for the function, which is pretty lengthy.

I would gladly remove the `useCallback` part of it so it's just a shorthand for getting the action from the context.

Reply:
Optimized this in fbdaaec8ba (and completely removed this file).